common/changes/office-ui-fabric-react/dropdown-types_2019-02-23-01-54.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Dropdown: `selectedKey` can now take null to clear the selection.",
```

Comment:
ComboBox is also affected by this change right?

Reply:
ComboBox doesn't inherit from Dropdown. Though ideally it ought to follow the same behavior.